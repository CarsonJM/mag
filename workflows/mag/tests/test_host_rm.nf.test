nextflow_workflow {

    name "Test workflow: MAG"
    script "../main.nf"
    workflow "MAG"
    tag "workflows"
    tag "mag"
    tag "mag_test_host_rm"
    config "../../../conf/mag_test_host_rm.config"

    test("Parameters: host_fasta != null") {

        then {
            assertAll(
                { assert workflow.success },
                { assert snapshot(
                    workflow.out,
                ).match()}
            )
        }
    }
}
