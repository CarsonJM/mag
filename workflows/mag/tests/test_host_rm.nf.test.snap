{
    "Parameters: host_fasta != null": {
        "content": [
            {
                "0": [
                    [
                        {
                            "id": "test_minigut_hg38host",
                            "group": "0",
                            "single_end": false
                        },
                        [
                            "test_minigut_hg38host_run0_host_removed.unmapped_1.fastq.gz:md5,48ab5cdbf8567cf6023bd304fcf896a9",
                            "test_minigut_hg38host_run0_host_removed.unmapped_2.fastq.gz:md5,b4246225aa4a15496991ac7c6b58563f"
                        ]
                    ],
                    [
                        {
                            "id": "test_minigut_sample2_hg38host",
                            "group": "0",
                            "single_end": false
                        },
                        [
                            "test_minigut_sample2_hg38host_run0_host_removed.unmapped_1.fastq.gz:md5,92e68dfac8156b01c547a26cb45e42b1",
                            "test_minigut_sample2_hg38host_run0_host_removed.unmapped_2.fastq.gz:md5,6ded2b5f426aca4e9b29877fac4e9bff"
                        ]
                    ]
                ],
                "1": [
                    
                ],
                "10": [
                    "versions.yml:md5,59050473c9f749b269ad7079fd4693df",
                    "versions.yml:md5,9e5858bca5fb800545dc5c11f8503cc4",
                    "versions.yml:md5,9e5858bca5fb800545dc5c11f8503cc4",
                    "versions.yml:md5,e9b46b38f7df41da088ff41127d3752a"
                ],
                "2": [
                    
                ],
                "3": [
                    
                ],
                "4": [
                    
                ],
                "5": [
                    
                ],
                "6": [
                    
                ],
                "7": [
                    
                ],
                "8": [
                    
                ],
                "9": [
                    
                ],
                "assemblies": [
                    
                ],
                "bin_summary": [
                    
                ],
                "cat": [
                    
                ],
                "genomad": [
                    
                ],
                "metaeuk": [
                    
                ],
                "prodigal": [
                    
                ],
                "prokka": [
                    
                ],
                "refined_bins": [
                    
                ],
                "refined_unbins": [
                    
                ],
                "short_reads": [
                    [
                        {
                            "id": "test_minigut_hg38host",
                            "group": "0",
                            "single_end": false
                        },
                        [
                            "test_minigut_hg38host_run0_host_removed.unmapped_1.fastq.gz:md5,48ab5cdbf8567cf6023bd304fcf896a9",
                            "test_minigut_hg38host_run0_host_removed.unmapped_2.fastq.gz:md5,b4246225aa4a15496991ac7c6b58563f"
                        ]
                    ],
                    [
                        {
                            "id": "test_minigut_sample2_hg38host",
                            "group": "0",
                            "single_end": false
                        },
                        [
                            "test_minigut_sample2_hg38host_run0_host_removed.unmapped_1.fastq.gz:md5,92e68dfac8156b01c547a26cb45e42b1",
                            "test_minigut_sample2_hg38host_run0_host_removed.unmapped_2.fastq.gz:md5,6ded2b5f426aca4e9b29877fac4e9bff"
                        ]
                    ]
                ],
                "versions": [
                    "versions.yml:md5,59050473c9f749b269ad7079fd4693df",
                    "versions.yml:md5,9e5858bca5fb800545dc5c11f8503cc4",
                    "versions.yml:md5,9e5858bca5fb800545dc5c11f8503cc4",
                    "versions.yml:md5,e9b46b38f7df41da088ff41127d3752a"
                ]
            }
        ],
        "timestamp": "2024-01-02T15:33:59.196829811"
    }
}